internal/backends/python/python.go

```diff
@@ -186,10 +214,15 @@ func pythonMakeBackend(name string, python string) api.LanguageBackend {
 			// be a pretty easy fix, though. (Why is this
 			// so complicated??)
 
-			outputB := util.GetCmdOutput([]string{
+			outputB, err := util.GetCmdOutputFallible([]string{
 				"poetry",
 				"config", "settings.virtualenvs.path",
 			})
+			if err != nil {
+				// there's no virtualenv configured, so no package directory
```

Comment:
Do we need to detect .pythonlibs here somehow? We weren't before and it was working.

Reply:
this is mostly to satisfy the test suite. i think something else is going on on replit, but `settings.virtualenvs.path` is broken with `.pythonlibs`
<img width="717" alt="image" src="https://github.com/replit/upm/assets/23486351/9ce297d7-99f7-4d3d-a89c-ce71e9b5d9fa">

i'll take the conversation about how to resolve this for nixmodules offline. i don't think packager3 ever actually calls this function, so i'm not too worried about it being at least slightly broken for the time being